components/audio_hal/Kconfig.projbuild

```diff
@@ -0,0 +1,19 @@
+menu "BOARD"
+choice BOARD
+    prompt "BOARD"
```

Comment:
Please don't use all-capitals for this text and the "menu" text. Suggest "Audio Board" not "BOARD"

Reply:
OK,Thanks for your advice.